lua/refactoring/treesitter/treesitter.lua

```diff
@@ -35,6 +38,35 @@ function TreeSitter:new(config, bufnr)
     return setmetatable(c, self)
 end
 
+function TreeSitter:isClassFunction(scope)
```

Comment:
something something, camel case is bad because something something sneks on a plane